content/en/blog/_posts/2021-04-08-PodSecurityPolicy-Past-Present-and-Future.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "PodSecurityPolicy: Past, Present, and Future"
+date: 2021-04-08
+slug: PodSecurityPolicy-Past-Present-and-Future
+---
+
+**Author:** Tabitha Sable (Kubernetes SIG Security)
+
+PodSecurityPolicy (PSP) is being deprecated in Kubernetes 1.21. This starts the countdown to its removal, but doesn’t change anything else. PodSecurityPolicy will continue to be fully functional for several more releases before being removed completely. In the meantime, we are developing a replacement for PSP that covers key use cases more easily and sustainably.
+
+What are Pod Security Policies? Why did we need them? Why are they going away, and what’s next? How does this affect you? These key questions come to mind as we prepare to say goodbye to PSP, so let’s walk through them together. We’ll start with an overview of how features get removed from Kubernetes.
+
+## What does deprecation mean in Kubernetes?
+
+Whenever a Kubernetes feature is set to go away, our [deprecation policy](docs/reference/using-api/deprecation-policy/) is our guide. First the feature is marked as deprecated, then after enough time has passed, it can finally be removed.
```

Comment:
:warning: broken hyperlink
```suggestion
Whenever a Kubernetes feature is set to go away, our [deprecation policy](/docs/reference/using-api/deprecation-policy/) is our guide. First the feature is marked as deprecated, then after enough time has passed, it can finally be removed.
```